### PR TITLE
Fixes some possession ability issues

### PR DESCRIPTION
**Pull request by Bartimeus973**
[bug]
## About the PR 
Possession is quite buggy for the trickster wraith. This is an attempt to fix wraith not retaking control of their body upon the death of the crewmate they were possessing.

Note: I dont think this fixes the mind issues when the body is gibbed yet. Help would be appreciatted.

## Why's this needed? 
Wraith being kicked out of the round when they did nothing wrong makes me sad. Probably makes them sad too.

### CODE REVIEW

File: code/datums/abilities/wraith.dm [hunk 1]
```diff
@@ -1327,19 +1327,20 @@ ABSTRACT_TYPE(/datum/targetable/wraithAbility/curse)
 	cooldown = 3 MINUTES
 	ignore_holder_lock = 0
 	var/wraith_key = null
+	var/datum/mind/wraith_mind = null
+	var/datum/mind/human_mind = null
```
Comment (ZeWaka) [inline code/datums/abilities/wraith.dm:1331]: please clear these on disposing

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/datums/abilities/wraith.dm (hunk 1 above)
Comment (Bartimeus973) [reply code/datums/abilities/wraith.dm:1331]: Donezo

Comment (ZeWaka) [reply code/datums/abilities/wraith.dm:1331]: Parent call should be flipped generally

Comment (Bartimeus973) [reply code/datums/abilities/wraith.dm:1331]: My bad! Done so.

**Pull request merged**